ksqldb-rest-app/src/main/java/io/confluent/ksql/rest/server/CommandTopicBackupImpl.java

```diff
@@ -89,7 +89,11 @@ public void initialize() {
       LOG.warn("Failed to read the latest backup from {}. Continue with a new file. Error = {}",
           replayFile.getPath(), e.getMessage());
 
-      replayFile = newReplayFile();
+      try {
+        replayFile = newReplayFile();
+      } catch (final IOException ee) {
+        throw new RuntimeException(ee);
```

Comment:
Is there any concern with adding new RuntimeExceptions?

Reply:
Nah, the upper layers need to be able to deal with runtime exceptions - and they do by throwing to the top and exiting the process.